tachyon/math/base/groups.h

```diff
@@ -151,8 +148,8 @@ class MultiplicativeGroup : public MultiplicativeSemigroup<G> {
     // Invert |product|.
     // (a₁ * a₂ * ... *  aₙ)⁻¹
     std::optional<G> product_inv_opt = product.Inverse();
-    if (UNLIKELY(
-            InvalidOperation(!product_inv_opt, "Inverse of zero attempted"))) {
+    if (UNLIKELY(!product_inv_opt)) {
```

Comment:
This case is exceptional for readability right?

Reply:
Yes I think so. This is a typical case that handles error.